app/bundles/CoreBundle/Command/GenerateProductionAssetsCommand.php

```diff
@@ -47,29 +43,59 @@ protected function configure()
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        // Check that the directory node_modules exists.
+        $nodeModulesDir = $this->pathsHelper->getSystemPath('root', true).'/node_modules';
```

Comment:
@escopecz this is the issue: depending on the way you installed mautic, the `node_modules` is not in the same dir.

this is basically the same issue as https://github.com/helios-ag/FMElfinderBundle/pull/499/files
And the gist of this fix can be used too.